docs/index.asciidoc

```diff
@@ -205,8 +206,18 @@ For more info, check out the
 The maximum amount of time, in seconds, to wait while establishing a connection to Elasticsearch.
 Connect timeouts tend to occur when Elasticsearch or an intermediate proxy is overloaded with requests and has exhausted its connection pool.
 
+[id="plugins-{type}s-{plugin}-custom_headers"]
+===== `custom_headers`
+
+  * Value type is <<hash,hash>>
+  * Default value is empty
+
+Pass a set of key value pairs as the headers sent in each request to an elasticsearch node.
+The headers will be used for any kind of request.
+These custom headers will be overridden by settings such as `compression_level`.
```

Comment:
This plugin doesn't have a `compression_level` setting, and it will override other headers since the setting is evaluated after others like user agent and authorization:

```suggestion
These custom headers will override any headers previously set by the plugin such as the User Agent or Authorization headers.
```